examples/tdec/simple_tdec.py

```diff
@@ -52,18 +57,19 @@
 message = "hello world".encode()
 ciphertext = enrico.encrypt_for_dkg(plaintext=message, conditions=eth_balance_condition)
 
-print(f'Encrypted message: {bytes(ciphertext).hex()}')
+print(f"Encrypted message: {bytes(ciphertext).hex()}")
 
 ###############
 # Bob
 ###############
-print('--------- Threshold Decryption ---------')
+print("--------- Threshold Decryption ---------")
 
 bob = Bob(
-    eth_provider_uri=provider_uri,
+    eth_provider_uri=staking_provider_uri,
```

Comment:
staking provider for Bob?

Reply:
Related to the above comment - I'm trying to indicate that this is the provider that nodes have their economic stake in.